src/components/Table/head/index.js

```diff
@@ -1,21 +1,62 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { uniqueId } from '../../../libs/utils';
+import Header from './header';
 
-export default function Head(props) {
-    const { columns } = props;
-    if (columns) {
-        return columns.map(column => (
-            <th key={uniqueId('header')} className="rainbow-table_header">{column.header}</th>
-        ));
+export default class Head extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            columnWidths: props.columns.map(column => column.width),
```

Comment:
what is the point of this?
it is only for change the column width?
if it is, then is better to not send a resize event to Head and instead change the width directly in the Header component with a width state.